examples/ColorCamera/rgb_undistort.py

```diff
@@ -0,0 +1,109 @@
+import time
+import cv2
+import depthai as dai
+import numpy as np
+
+resolutionMap = {"THE_1080_P": (1920, 1080), "THE_800_P": (1280, 800), "THE_720_P": (1280, 720), "THE_400_P": (640, 400)}
+resolution = resolutionMap["THE_1080_P"]
```

Comment:
rather use `dai.ColorCameraProperties.SensorResolution.THE_1080_P` as key

Reply:
e.g. 
```
camResolution = dai.ColorCameraProperties.SensorResolution.THE_1080_P
resolutionMap[camResolution]
```
so it can be set easily by changing just the variable